waku/waku.md

```diff
@@ -163,8 +164,7 @@ packet-format = "[" packet-code packet-format "]"
 
 required-packet = 0 status /
                   1 messages /
-		  2 pow-requirement /
-		  3 bloom-filter
+		  2 status-update /
```

Comment:
Still need to add the `status-update` higher up. Basically containing just `status-options`.